### PR TITLE
Revamp Zuul job config

### PR DESCRIPTION
**Pull request by ssbarnea**

### CODE REVIEW

**Comment by apatard** [comment]
I'm not convinced it's a good idea. Things are blocked by CI issues and this sucks. This commit would help but I fear that it will allow merging broken code. I mean, from what I understand, with luck the job may run on one of the few hosts with nested virt, detect a problem with the PR but the code will still be merged.
I've also thought about trying to switch to using libvirt+qemu with kvm disabled but I'm sure it'll work and it's possibly going to be dead slow...

**Comment by ssbarnea** [comment]
> I'm not convinced it's a good idea. Things are blocked by CI issues and this sucks. This commit would help but I fear that it will allow merging broken code. I mean, from what I understand, with luck the job may run on one of the few hosts with nested virt, detect a problem with the PR but the code will still be merged.
> I've also thought about trying to switch to using libvirt+qemu with kvm disabled but I'm sure it'll work and it's possibly going to be dead slow...

Doing a fallback to quemu would be great.

**Comment by apatard** [comment]
I've a begging of a solution to fallback to qemu. I'm still testing it. Once done, I'll update PR44

**Comment by ssbarnea** [comment]
recheck

**Comment by ssbarnea** [comment]
recheck

File: tools/test-setup.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@ if [ -f /etc/os-release ]; then
                 18.04)
                     # ubuntu xenial vagrant is too old so it doesn't support triggers, used by the alpine box
                     sudo apt-get remove --purge -y vagrant
-                    wget https://releases.hashicorp.com/vagrant/2.2.9/vagrant_2.2.9_x86_64.deb
+                    wget --no-show-progress https://releases.hashicorp.com/vagrant/2.2.9/vagrant_2.2.9_x86_64.deb
```
Comment (apatard) [inline tools/test-setup.sh:44]: btw, maybe update to 2.2.10 here too, as done with rpm distros ?

Comment (ssbarnea) [reply tools/test-setup.sh:44]: Can be done later, this already brew more than needed.

**Pull request merged**